ingest/queries/describeDataset.rq

```diff
@@ -2,23 +2,28 @@ PREFIX rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
 PREFIX rdfs:  <http://www.w3.org/2000/01/rdf-schema#>
 PREFIX xsd:   <http://www.w3.org/2001/XMLSchema#>
 PREFIX owl:   <http://www.w3.org/2002/07/owl#>
+PREFIX dcat: <http://www.w3.org/ns/dcat#>
+PREFIX dct: <http://purl.org/dc/terms/>
 PREFIX dco: <http://info.deepcarbon.net/schema#>
+PREFIX dcodata: <http://info.deepcarbon.net/data/schema#>
 PREFIX dc: <http://purl.org/dc/elements/1.1/>
 PREFIX dct: <http://purl.org/dc/terms/>
 PREFIX vivo: <http://vivoweb.org/ontology/core#>
 PREFIX vitro: <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#>
 PREFIX foaf: <http://xmlns.com/foaf/0.1/>
 PREFIX bibo: <http://purl.org/ontology/bibo/>
+PREFIX prov: <http://www.w3.org/ns/prov#>
 
-DESCRIBE ?dataset ?dcoid ?mostSpecificType ?label ?publicationYear ?community ?team ?project ?authorship ?author ?researchArea ?orgrole ?org ?interval ?end ?dataType ?citation ?distribution ?file ?fileURL
+DESCRIBE ?dataset ?dcoid ?mostSpecificType ?publicationYear ?community ?team ?project ?authorship ?author ?researchArea ?orgrole ?org ?interval ?end ?dataType ?citation ?distribution
 WHERE {
   ?dataset dco:hasDcoId ?dcoid .
   ?dataset vitro:mostSpecificType ?mostSpecificType .
   ?dataset rdfs:label ?label .
-  OPTIONAL { ?dataset dco:yearOfPublication ?publicationYear . }
+  OPTIONAL { ?dataset dct:description ?description . }
+  OPTIONAL { ?dataset dct:issued ?publicationYear . }
```

Comment:
?publicationYear is a datatype and does not need to be included in the query.  Since we are doing a DESCRIBE on ?dataset all the datatype property values for each ?dataset will by default be in the return graph.